pallets/multisig/src/benchmarking.rs

```diff
@@ -0,0 +1,284 @@
+
+    remove_multisig_signer {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build_with_did("bob", 0);
+        let signers = vec![
+            Signatory::from(alice.did()),
+            Signatory::from(bob.did()),
+        ];
+        let multisig = generate_multisig_with_signers::<T>(alice.account(), alice.origin(), signers, 1)?;
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, Signatory::from(bob.did()))
+
+    add_multisig_signers_via_creator {
+        // Number of signers
+        let i in 1 .. T::MaxSigners::get();
+
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::from(alice.did())], 1)?;
+        let mut signers = vec![];
+        generate_signers::<T>(&mut signers, i as usize);
+    }: _(alice.origin, multisig, signers)
+
+    remove_multisig_signers_via_creator {
+        // Number of signers
+        let i in 1 .. T::MaxSigners::get();
+
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build_with_did("bob", 0);
+        let signers = vec![
+            Signatory::from(alice.did()),
+            Signatory::from(bob.did()),
+        ];
+        let multisig = generate_multisig_with_signers::<T>(alice.account(), alice.origin(), signers, 1)?;
+        let signers = vec![
+            Signatory::from(bob.did()),
+        ];
+    }: _(alice.origin, multisig, signers)
+
+    change_sigs_required {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let signers = vec![Signatory::from(alice.did())];
+        let multisig = generate_multisig::<T>(alice.account(), alice.origin(), signers)?;
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, 1)
+
+    change_all_signers_and_sigs_required {
+        // Number of signers
+        let i in 1 .. T::MaxSigners::get();
+
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let multisig = generate_multisig_with_signers::<T>(alice.account(), alice.origin(), vec![Signatory::from(alice.did())], 1)?;
+        let mut signers = vec![];
+        generate_signers::<T>(&mut signers, i as usize);
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, signers, 1)
+
+    make_multisig_signer {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::from(alice.did())], 1)?;
+        let multisig_new = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::from(alice.did())], 1)?;
```

Comment:
Why are we creating 2 multisigs here?

Reply:
I forgot to remove that after the chat we had clearing up what that extrinsic was doing.